salt/modules/cmdmod.py

```diff
@@ -2643,8 +2643,7 @@ def script(
             salt myminion cmd.script salt://foo.sh "arg1 'arg two' arg3"
 
     :param str cwd: The directory from which to execute the command. Defaults
-        to the home directory of the user specified by ``runas`` (or the user
-        under which Salt is running if ``runas`` is not specified).
+        to the directory returned from Python's tempfile.mkstemp.
```

Comment:
can you help my understanding and point in the code where this is happening?

Reply:
https://github.com/saltstack/salt/blob/2a030cd3f89966181f12d94bb863f89812e09813/salt/modules/cmdmod.py#L2830 is where it's passing cwd, which heads to https://github.com/saltstack/salt/blob/2a030cd3f89966181f12d94bb863f89812e09813/salt/utils/files.py#L101

Those docs were a bit of a lie - https://docs.python.org/3/library/tempfile.html#tempfile.mkstemp is how mkstemp *actually* works.